openedx/openedx-platform
# Hackathon project: burn coffee

Description: Removes discussion-related coffee files.
### Testing instructions:

Smoke test:
1. Create course/section/subsection/etc.
2. Add Discussion module.
3. Log in into LMS.
4. Open the unit with discussion.
5. Click "Show discussion". Expected result: empty discussion is shown, but no errors are in console.
6. Go wild: create and edit threads, responses and comments, upvote/downvote, report, edit, etc.
7. Make sure nothing blows in your face (i.e. does what it is supposed to do, no errors in browser console, etc.)
### Sandbox

https://coffee-to-js-discussions.sandbox.edx.org/
### Reviewers:
- [x] @andy-armstrong 
- [x] @bjacobel
### Post-review
